webrtc/webapp/src/main/webapp/WEB-INF/conf/configuration.xml

```diff
@@ -0,0 +1,299 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+
+  Copyright (C) 2017 eXo Platform SAS.
+  
+  This is free software; you can redistribute it and/or modify it
+  under the terms of the GNU Lesser General Public License as
+  published by the Free Software Foundation; either version 2.1 of
+  the License, or (at your option) any later version.
+  
+  This software is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  Lesser General Public License for more details.
+  
+  You should have received a copy of the GNU Lesser General Public
+  License along with this software; if not, write to the Free
+  Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  02110-1301 USA, or see the FSF site: http://www.fsf.org.
+
+-->
+
+<configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_3.xsd http://www.exoplatform.org/xml/ns/kernel_1_3.xsd"
+  xmlns="http://www.exoplatform.org/xml/ns/kernel_1_3.xsd">
+  
+  <!-- WebRTC provider plugin -->
+  <external-component-plugins>
+    <target-component>org.exoplatform.webconferencing.WebConferencingService</target-component>
+    <component-plugin>
+      <name>add.clouddriveprovider</name>
+      <set-method>addPlugin</set-method>
+      <type>org.exoplatform.webconferencing.webrtc.WebrtcProvider</type>
+      <init-params>
+        <properties-param>
+          <name>provider-configuration</name>
+          <property name="disable" value="${webconferencing.webrtc.disable:false}" />

```

Comment:
This disable property seems to be never used. Shouldn't we remove provider-configuration properties ?